.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "destiny-icons"]
 	path = destiny-icons
 	url = https://github.com/justrealmilk/destiny-icons
+[submodule "d2-additional-info-module"]
+	path = d2-additional-info-module
```

Comment:
Could you just submodule this directly into the data directory?

Reply:
:+1: